snippets/C#/modelprop.snippet

```diff
@@ -37,6 +37,11 @@
                 <![CDATA[/// <summary>
 /// $description$.
 /// </summary>
+#if NETFX_CORE
+[DataMember]
```

Comment:
To be honest I don't think you need to define DataMember. You can just use the Catel serializers (binary, xml and json) to serialize any model.